synapse/lib/rstorm.py

```diff
@@ -235,7 +234,7 @@ async def _handleStorm(self, text):
         cli = await StormCliOutput.anit(item=core, outp=outp)
 
         try:
-            args = s_parser.Parser(text).cmdargs()
+            args = shlex.split(text)
```

Comment:
Can we add a small test that demonstrates the need for this behavior? That way we can account for future changes we may need to make or consider.